test/common.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+global.Promise    = require('bluebird');
```

Comment:
Why do you need bluebird?

Reply:
i.e. Can't you just use ES6 promises?

Reply:
I was using `bluebird` for testing, but no longer, so it's gone...